CMakeLists.txt

```diff
@@ -64,14 +64,20 @@ option(ENABLE_GLES          "Build for OpenGL ES 2.0 instead of OpenGL 2.1 (Defa
 option(USE_GTKGLEXT         "Use libgtkglext1 for GTK2 frontend (Default: on)" ON)
 option(USE_QT6              "Use Qt6 in Qt frontend (Default: off)" OFF)
 option(USE_GTK3             "Use Gtk3 in GTK2 frontend (Default: off)" OFF)
-option(USE_WAYLAND          "Use Wayland in Qt frontend (Default: off)" OFF)
```

Comment:
USE_WAYLAND option is gone again?

Reply:
oops, I think I’ve deleted this line by accident.

Reply:
I don’t have access to a PC now, so if anyone can help add it back, feel free to do so.

Reply:
oops! :) i returned it back